snuba/settings/validation.py

```diff
@@ -117,3 +117,49 @@ def validate_slicing_settings(locals: Mapping[str, Any]) -> None:
         assert (
             topic_tuple in locals["SLICED_KAFKA_BROKER_CONFIG"]
         ), f"missing broker config definition for sliced Kafka topic {topic_tuple[0]} on slice {topic_tuple[1]}"
+
+    _STORAGE_SET_CLUSTER_MAP = {
+        storage_set: cluster
+        for cluster in locals["CLUSTERS"]
+        for storage_set in cluster["storage_sets"]
+    }
+
+    _SLICED_STORAGE_SET_CLUSTER_MAP: Dict[Tuple[str, int], Mapping[str, Any]] = {}
```

Comment:
nit

```
Dict[Tuple[str, int], Mapping[str, Any]]
```

Is a quite hard type to read and wrap one's mind around. Adding aliases to give some context would be helpful. Also please `MutableMapping` instead of Dict as it is more generic.

Reply:
Ok, modified!